packages/atomic/cypress/e2e/smart-snippet-assertions.ts

```diff
@@ -56,32 +49,27 @@ export function assertAnswerBottomMargin(
     SmartSnippetSelectors.answer()
       .find(`.${lastElementClass}`)
       .distanceTo(SmartSnippetSelectors.footer)
-      .should('have.property', 'vertical', margin);
+      .should('have.property', 'vertical')
+      .and('be.closeTo', margin, 0.1);
   });
 }
 
 export function assertLikeButtonChecked(checked: boolean) {
```

Comment:
I feel like this is more appropriate:
```suggestion
export function assertLikeButtonCheck(checked: boolean) {
```

`Checked` implies that it's already checked (aka active), but this can test both checked and unchecked 🙂